docs/tutorials/java/ssd_inference.md

```diff
@@ -0,0 +1,182 @@
+# Multi Object Detection using pre-trained SSD Model via Java Inference APIs
+
+This tutorial shows how to use MXNet Java Inference APIs to run inference on a pre-trained Single Shot Detection (SSD) Model.
+
+The SSD model is trained on the Pascal VOC 2012 dataset. The network is a SSD model built on Resnet50 as base network to extract image features. The model is trained to detect the following entities (classes): ['aeroplane', 'bicycle', 'bird', 'boat', 'bottle', 'bus', 'car', 'cat', 'chair', 'cow', 'diningtable', 'dog', 'horse', 'motorbike', 'person', 'pottedplant', 'sheep', 'sofa', 'train', 'tvmonitor']. For more details about the model, you can refer to the [MXNet SSD example](https://github.com/apache/incubator-mxnet/tree/master/example/ssd).
+
+## Pre-Requisites
+
+To complete this tutorial, we need : 
+* [MXNet Java Setup on IntelliJ IDEA](/java/mxnet_java_on_intellij.html) 
+* [wget](https://www.gnu.org/software/wget/) To download model artifacts 
+* SSD Model artifacts
+    * Use the following script to get the SSD Model files : 
+    ```bash
+      data_path=/tmp/resnet50_ssd
+      mkdir -p "$data_path"
+      wget https://s3.amazonaws.com/model-server/models/resnet50_ssd/resnet50_ssd_model-symbol.json -P $data_path
+      wget https://s3.amazonaws.com/model-server/models/resnet50_ssd/resnet50_ssd_model-0000.params -P $data_path
+      wget https://s3.amazonaws.com/model-server/models/resnet50_ssd/synset.txt -P $data_path
+    ```
+* Dataset  : A few sample images to run inference on.
+    * Use the following script to download sample images : 
```

Comment:
Are we providing these scripts on the example folders? Copy pasting gets tiring and is error prone, meaning docs can get outdated and not run anymore, is easier to keep scripts current.

Reply:
Yes they are in the examples folder already. 
https://github.com/apache/incubator-mxnet/blob/master/scala-package/examples/scripts/infer/objectdetector/get_ssd_data.sh

Reply:
It's a hands-on tutorial, where we want users to follow the commands on the same page. 
While you have a valid point of re-directing to the link, but it's not a great user experience to hop urls while going though a hands on session. 
Also, the commands are not verbatim pasted. Only a relevant subset of wget commands are pasted here in the tutorial.

Reply:
ok, thanks for the followup.